### DISCUSSION TITLE
WinUI 3 docs pages are broken

### DISCUSSION THREAD

**Discussion by jaigak** [microsoft/WindowsAppSDK#2561] [General] [answered]
Every WinUI 3 docs page seems to show a 404 - Page not found error now.

**Comment by castorix** [comment]
Which WinUI pages ?
I have actually no problem with links like
 https://docs.microsoft.com/en-us/windows/apps/winui/
https://docs.microsoft.com/en-us/windows/apps/winui/winui3/desktop-winui3-app-with-basic-interop
https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/
etc...

**Reply by TrebuchKill** [reply]
As I seem to encounter the same problem using google just now, I'll provide an example:
https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.application.resources
https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.resourcedictionary

**Reply by jaigak** [reply]
It only seems to be the case for API reference links, tutorials just load fine.

**Comment by kmgallahan** [comment, marked as answer]
https://github.com/MicrosoftDocs/winui-api/issues/119

https://github.com/microsoft/microsoft-ui-xaml/issues/7157
